script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -119,7 +119,7 @@ Tohno Shiki is to return to the Tohno household at once.'
 [sha:04b1459f61c4e8d1ad80da0e20b06ed713e52cc8]{
 -- Page 9, Offset 1169.
 -- 七年近く音沙汰のなかった実家からの電報は、そんな意味合いのものだったという。
-The telegram sent from my original household, whom I hadn't heard from in nearly seven years, said something to that effect.
+The telegram sent from my original household, from whom I hadn't heard from in nearly seven years, said something to that effect.
```

Comment:
still feels off to me

Reply:
i still don't agree, but the old version isn't that bad so i'll revert